pages/users/[id].tsx

```diff
@@ -154,7 +159,12 @@ export default function User({ loginContext }: Props) {
 
   // Recent Activity hooks
   const { $hasPrevious, $hasNext, fetchPrevious, fetchNext } =
-    usePaginatedEvents(userId, EVENTS_LIMIT, $events, $setEvents)
+    usePaginatedEvents(
+      userId,
+      EVENTS_LIMIT,
+      $events as API.ListEventsResponse,
```

Comment:
I think this `as` is removable, the build seems to pass without it.

In general, we should avoid using `as` where possible since it punches a hole in the type system. (you could do `as number` even though it's not actually that)

Reply:
It was required in the very first version, but now it is passing without this cast.